arch/x86/x86_64/include/uk/asm/paging.h

```diff
@@ -187,6 +194,16 @@ static inline int ukarch_vaddr_range_isvalid(__vaddr_t start, __vaddr_t end)
 #define PAGE_ATTR_PROT_WRITE		0x02 /* Page is writeable */
 #define PAGE_ATTR_PROT_EXEC		0x04 /* Page is executable */
 
+/* Page fault error code bits */
+#define X86_PF_EC_P			0x0001UL /* 0=non-present, 1=prot */
+#define X86_PF_EC_WR			0x0002UL /* 0=read, 1=write */
+#define X86_PF_EC_US			0x0004UL /* 0=kernel, 1=user */
+#define X86_PF_EC_RSVD			0x0008UL /* reserved bit in PTE */
+#define X86_PF_EC_ID			0x0010UL /* instruction fetch */
+#define X86_PF_EC_PK			0x0020UL /* protection key violation */
+#define X86_PF_EC_SS			0x0040UL /* shadow stack access */
```

Comment:
```suggestion
#define X86_PF_EC_SS			0x0040UL /* shadow stack access */
#define X86_PF_EC_HLAT			0x0080UL /* no translation using HLAT paging */
```
Or is there a reason to that one bit?